src/components/faceVerification/standalone/utils/giveupReason.js

```diff
@@ -1,38 +1,34 @@
 import { t } from '@lingui/macro'
 
-export const GiveUpReason = new class {
-  CameraIssue = 'camera'
-
-  OvalIssue = 'oval'
-
-  TwinIssue = 'twin'
-
-  TryAgainIssue = 'tryAgain'
+const cancelledOptions = {
+  noScan: t`I don't want my face scanned`,
+  noTime: t`I don't have time for this`,
+  noComprendo: t`I don't understand what I'm doing`,
+  doLater: t`I will do it later`,
+  techIssues: t`I'm having technical issues`,
+  noVerify: t`I don't want to GoodDollar-verify anymore`,
+}
+
+const failedOptions = {
+  NotRecognizeFace: t`My face is not recognized on the camera`,
+  LowCameraQuali: t`I think my camera quality is too low`,
+  OvalIssue: t`Unable to put my face in the Oval`,
+  TwinIssue: t`It says I have a twin`,
+  Privacy: t`Privacy concern`,
+  BadInternet: t`I have a bad internet connection`,
+  DoItLater: t`I will do it later`,
+}
```

Comment:
```suggestion
const GiveUpCancelled = {
  NoScan: t`I don't want my face scanned`,
  NoTime: t`I don't have time for this`,
  NoComprendo: t`I don't understand what I'm doing`,
  DoLater: t`I will do it later`,
  TechIssues: t`I'm having technical issues`,
  NoVerify: t`I don't want to GoodDollar-verify anymore`,
}

const GiveUpFailed = {
  NotRecognizeFace: t`My face is not recognized on the camera`,
  LowCameraQuali: t`I think my camera quality is too low`,
  OvalIssue: t`Unable to put my face in the Oval`,
  TwinIssue: t`It says I have a twin`,
  Privacy: t`Privacy concern`,
  BadInternet: t`I have a bad internet connection`,
  DoItLater: t`I will do it later`,
}
```

I think it would be enough.
just pass the corresponding object to useGiveupDialog, call Object.entries at OptionsRow render

Reply:
ye you are right, these object are enough. Removed old giveUp implementation